Components/Avatar/avatar.css

```diff
@@ -0,0 +1,31 @@
+@import url('../../styles/utlity.css');
+@import url('../../styles/navBar.css');
+@import url('../../styles/sideBar.css');
+@import url('../../styles/variable.css');
+.avatar{
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
```

Comment:
Try to use relative units instead of absolute units.

Reply:
made neccesary changes...thanx